recipes/sdl_image/all/conanfile.py

```diff
@@ -120,10 +138,10 @@ def generate(self):
         tc.variables["XPM"] = self.options.xpm
         tc.variables["XV"] = self.options.xv
         # TODO: https://github.com/bincrafters/community/pull/1317#pullrequestreview-584847138
-        tc.variables["TIF_DYNAMIC"] = self.dependencies["libtiff"].options.shared if self.options.with_libtiff else False
-        tc.variables["JPG_DYNAMIC"] = self.dependencies["libjpeg"].options.shared if self.options.with_libjpeg else False
-        tc.variables["PNG_DYNAMIC"] = self.dependencies["libpng"].options.shared if self.options.with_libpng else False
-        tc.variables["WEBP_DYNAMIC"] = self.dependencies["libwebp"].options.shared if self.options.with_libwebp else False
+        tc.variables["TIF_DYNAMIC"] = self.options.get_safe("with_libtiff_dynamic_load", False)
```

Comment:
The upstream variables have the format `SDL2IMAGE_TIF` and `SDL2IMAGE_TIF_SHARED`, etc.

Reply:
is it ok with https://github.com/conan-io/conan-center-index/pull/14493/commits/657257333afba783024f36eecf6cc44d7b391359?